stream-chat-android/src/main/java/com/getstream/sdk/chat/utils/CollectionsUtils.kt

```diff
@@ -0,0 +1,34 @@
+package com.getstream.sdk.chat.utils
+
+// Use when it is known that the element is always close to the end
+internal fun <T> List<T>.reverseFind(predicate: (T) -> Boolean): T? {
```

Comment:
Isn't it the same like the Kotlin extension `lastOrNull()`?
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/last-or-null.html

Reply:
Also the same as [`findLast`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/find-last.html), which has a better fitting name for here

Reply:
True. I never checked the implementation of `findLast`... I though it did a search increasing the index instead of decreasing it.